core-services/prow/02_config/generate-boskos.py

```diff
@@ -43,6 +43,13 @@
         'us-east-1': 25,
         'ap-northeast-1': 5,
     },
+    'aws-terraform-qe-quota-slice': {
+        'ap-northeast-1': 2,
+        'us-east-1': 2,
+        'us-east-2': 2,
+        'us-west-1': 2,
+        'us-west-2': 2,
```

Comment:
I assume `2` is small for usage. The number at least > 3 for one is automation, another one is debugging, the third one is for any accident.